fmriprep/cli/run.py

```diff
@@ -345,11 +375,15 @@ def main():
             errno = 1
         else:
             raise
-
-    # Generate reports phase
-    errno += generate_reports(subject_list, output_dir, work_dir, run_uuid)
-    write_derivative_description(bids_dir, str(Path(output_dir) / 'fmriprep'))
-    sys.exit(int(errno > 0))
+    finally:
+        # Generate reports phase
+        errno += generate_reports(subject_list, output_dir, work_dir, run_uuid,
+                                  sentry_sdk=sentry_sdk)
+        write_derivative_description(bids_dir, str(Path(output_dir) / 'fmriprep'))
+
+        if not opts.notrack and errno == 0:
+            sentry_sdk.capture_message('fMRIPrep finished without errors', level='info')
```

Comment:
```suggestion
            sentry_sdk.capture_message('fMRIPrep finished with exit code %d' % errno, level='info')
```

Reply:
Exit code in fmriprep corresponds to the numebr of errors not their category. Addind this message to sentry mesages will make grouping messages about failed runs very difficult. Not sure I see the use.